packages/terra-clinical-data-grid/src/subcomponents/Row.jsx

```diff
@@ -87,6 +94,7 @@ class Row extends React.Component {
         data-row
         data-row-id={rowId}
         data-section-id={sectionId}
+        data-allow-column-highlight={(!isSelected && allowColumnHighlighting) ? true : null}
```

Comment:
I think just `allowColumnHighlighting` would be enough to set value for data-attr. Since `allowColumnHighlighting` value is already dependent on row.isSelected in DataGrid.jsx .

Reply:
Correct, We would not need that here. [Updated](https://github.com/cerner/terra-clinical/pull/796/commits/d61cc802366b240b410fd07f004d522d32624e81#diff-752ea2553f0e37a1f464aa5f9d0f6fa2fd919ddd91e8bc87a02945878fadc53fR97)